robocop/utils/disablers.py

```diff
@@ -74,14 +66,16 @@ def _parse_file(self, source):
 
     def _parse_line(self, line, lineno):
         statement, comment = line.split('#', maxsplit=1)
+        if '# noqa' in line:
```

Comment:
What about `#noqa`? Shouldn't it be supported?

Reply:
We could support it but it would be rule volation - comments needs space after # (missing-space-after-comment)